items.go

```diff
@@ -53,7 +53,7 @@ type itemType int
 
 // These items define the UCI language.
 const (
-	itemError itemType = iota // error occurred; item.val is text of error
+	itemError itemType = iota // errstr occurred; item.val is text of errstr
```

Comment:
Global search+replace is seldomly a good idea :)

This should still read

```suggestion
	itemError itemType = iota // error occurred; item.val is error message
```

or similar.

Reply:
Oh. This was Goland doing refactoring (which is kinda like a search/replace). Mea culpa.